pkg/metrics/instance/instance.go

```diff
@@ -427,7 +428,15 @@ func (i *Instance) initialize(ctx context.Context, reg prometheus.Registerer, cf
 	i.storage = storage.NewFanout(i.logger, i.wal, i.remoteStore)
 
 	opts := &scrape.Options{
-		ExtraMetrics: cfg.global.ExtraMetrics,
+		ExtraMetrics:      cfg.global.ExtraMetrics,
+		HTTPClientOptions: []config_util.HTTPClientOption{},
```

Comment:
tip: You don't have to set this to an empty slice since it's valid to append to a nil slice, though doing this makes it more clear that it should be empty to start with.